<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图书管理系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="bookTab">
       <attribute name="title">
        <string>图书管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="bookTabLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="bookToolbarLayout">
          <item>
           <widget class="QLineEdit" name="bookSearchEdit">
            <property name="placeholderText">
             <string>搜索：ISBN、书名、作者...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="bookCategoryLabel">
            <property name="text">
             <string>分类:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="bookCategoryCombo">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bookAddBtn">
            <property name="text">
             <string>添加</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bookEditBtn">
            <property name="text">
             <string>编辑</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bookDeleteBtn">
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="bookTableView">
          <property name="selectionMode">
           <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="readerTab">
       <attribute name="title">
        <string>读者管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="readerTabLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="readerToolbarLayout">
          <item>
           <widget class="QLineEdit" name="readerSearchEdit">
            <property name="placeholderText">
             <string>搜索：读者编号、姓名或电话...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="readerAddBtn">
            <property name="text">
             <string>添加</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="readerEditBtn">
            <property name="text">
             <string>编辑</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="readerDeleteBtn">
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="readerTableView">
          <property name="selectionMode">
           <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="borrowTab">
       <attribute name="title">
        <string>借还书管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="borrowTabLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="borrowToolbarLayout">
          <item>
           <widget class="QLineEdit" name="borrowSearchEdit">
            <property name="placeholderText">
             <string>搜索：读者编号或图书ISBN...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="borrowBookBtn">
            <property name="text">
             <string>借书</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="returnBookBtn">
            <property name="text">
             <string>还书</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="borrowTableView">
          <property name="selectionMode">
           <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="statisticsTab">
       <attribute name="title">
        <string>数据统计</string>
       </attribute>
       <layout class="QVBoxLayout" name="statisticsTabLayout">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <layout class="QGridLayout" name="statisticsGridLayout">
          <property name="spacing">
           <number>15</number>
          </property>
          <item row="0" column="0">
           <widget class="QWidget" name="totalBooksCard" native="true">
            <property name="styleSheet">
             <string>QWidget { background-color: #f5f5f5; border-radius: 5px; padding: 15px; }</string>
            </property>
            <layout class="QVBoxLayout" name="totalBooksLayout">
             <item>
              <widget class="QLabel" name="totalBooksTitleLabel">
               <property name="styleSheet">
                <string>color: #666; font-size: 14px;</string>
               </property>
               <property name="text">
                <string>图书总数</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="totalBooksLabel">
               <property name="styleSheet">
                <string>color: #333; font-size: 24px; font-weight: bold;</string>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QWidget" name="totalReadersCard" native="true">
            <property name="styleSheet">
             <string>QWidget { background-color: #f5f5f5; border-radius: 5px; padding: 15px; }</string>
            </property>
            <layout class="QVBoxLayout" name="totalReadersLayout">
             <item>
              <widget class="QLabel" name="totalReadersTitleLabel">
               <property name="styleSheet">
                <string>color: #666; font-size: 14px;</string>
               </property>
               <property name="text">
                <string>读者总数</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="totalReadersLabel">
               <property name="styleSheet">
                <string>color: #333; font-size: 24px; font-weight: bold;</string>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QWidget" name="currentBorrowsCard" native="true">
            <property name="styleSheet">
             <string>QWidget { background-color: #f5f5f5; border-radius: 5px; padding: 15px; }</string>
            </property>
            <layout class="QVBoxLayout" name="currentBorrowsLayout">
             <item>
              <widget class="QLabel" name="currentBorrowsTitleLabel">
               <property name="styleSheet">
                <string>color: #666; font-size: 14px;</string>
               </property>
               <property name="text">
                <string>当前借出</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="currentBorrowsLabel">
               <property name="styleSheet">
                <string>color: #333; font-size: 24px; font-weight: bold;</string>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QWidget" name="overdueCountCard" native="true">
            <property name="styleSheet">
             <string>QWidget { background-color: #ffe6e6; border-radius: 5px; padding: 15px; }</string>
            </property>
            <layout class="QVBoxLayout" name="overdueCountLayout">
             <item>
              <widget class="QLabel" name="overdueCountTitleLabel">
               <property name="styleSheet">
                <string>color: #666; font-size: 14px;</string>
               </property>
               <property name="text">
                <string>逾期数量</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="overdueCountLabel">
               <property name="styleSheet">
                <string>color: #d32f2f; font-size: 24px; font-weight: bold;</string>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="statisticsSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
